packages/cli-kit/src/path.ts

```diff
@@ -37,9 +37,11 @@ export async function findUp(
  * @param path - Path to relativize
  * @returns Relativized path.
  */
-export function relativize(path: string): string {
-  const result = commondir([path, process.cwd()])
-  const relativePath = relative(process.cwd(), path)
+export function relativize(path: string, cwd: string = process.cwd()): string {
+  const result = commondir([path, cwd])
+  const relativePath = relative(cwd, path)
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
   const relativeComponents = relativePath.split('/').filter((component) => component === '..').length
```

Comment:
`pathe` imports Node modules using the `node:....` module ID, which is not recognized by the version 
 of `@types/node` that we are using and therefore Typescript recognizes some of the types as `any`. This mitigates the issue until we can drop support for Node 14 and update `@types/node`.